WcaOnRails/spec/support/database_cleaner.rb

```diff
@@ -1,22 +1,18 @@
 RSpec.configure do |config|
   config.before(:suite) do
-    DatabaseCleaner.clean_with(:truncation)
+    DatabaseCleaner.clean_with(:truncation, except: %w(Countries Continents Events Rounds Formats teams))
```

Comment:
This is a DRY problem. We should use the same list of classes/tables here as in `spec_helper`, not a copy.